_wiki/essential-knowledge/working-with-futures.md

```diff
@@ -11,7 +11,7 @@ keywords:
 
 > This tutorials assumes, you are familiar with lambda expressions. Take a look at the [Lambda Introduction](/wiki/essential-knowledge/lambda-introduction) first, if you are not!
 
-As Javacord is heavily multithreaded, you have to understand the concept of [Futures](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/Future.html) in general, as well as their most common implementation, the [CompletableFuture](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletableFuture.html). This little introduction should give you a quick overview about the basics you need to know in order to work with Futures.
+As Javacord is heavily multithreaded, you have to understand the concept of [Futures](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/Future.html) in general, as well as their most common implementation, the [CompletableFuture](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletableFuture.html). This little introduction should give you a quick overview of the basics you need to know in order to work with Futures.
```

Comment:
Is this really correct?
https://ell.stackexchange.com/questions/11583/provides-information-on-of-or-about-somthing

Reply:
I believe so, although after looking over it, both are correct. The key difference is that in that link, `provides information` is essentially a verb, which `on`, `of`, and `about` operate on, but in the wiki `a quick overview` is essentially a noun. `an overview of` taken out of context is correct, as far as I can tell, and is used more often in regular language than `an overview about`. So, it's up to you which you want to use.

Reply:
Let's go with `of` then, if you think that's more common.